pkg/asset/cluster/tfvars.go

```diff
@@ -73,6 +74,12 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 	rhcosImage := new(rhcos.Image)
 	parents.Get(clusterID, installConfig, bootstrapIgnAsset, masterIgnAsset, mastersAsset, rhcosImage)
 
+	platform := installConfig.Config.Platform.Name()
+	switch platform {
+	case none.Name, vsphere.Name:
```

Comment:
AWS can also be UPI, do we need something else to denote that it's IPI/UPI?

Reply:
currently the UPI for AWS is same as IPI for installer. except that user decided `create ignition-configs` and stopping there. If we do end up making changes (@wking was looking at that.) we can come back.